scripts/python/docs-automation/sync_tocs.py

```diff
@@ -147,6 +182,14 @@ def get_non_standard_hrefs(doc_repo_location):
         default="",
         required=False,
     )
+    
+    parser.add_argument(
```

Comment:
We have to pull these documents from different folders now right. So instead of 

`docs-ref-services/storage-blob-readme-pre.md`

we have

`docs-ref-services/preview/storage-blob-readme.md`

The fact that these readmes are identically named enables us to have flawless swap between `overviews`.

Reply:
Thanks! Got the idea from another PR. The structure is much cleaner than before. Great improvement.